gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/internal/PluginRegistryImpl.java

```diff
@@ -69,22 +71,26 @@ public class PluginRegistryImpl extends AbstractService implements PluginRegistr
     }
 
     @Autowired
+    @Setter(AccessLevel.PACKAGE)
```

Comment:
I suppose you did that for test purpose, best way is to use a constructor. See for example https://reflectoring.io/constructor-injection/